Dockerfile

```diff
@@ -18,7 +18,8 @@ RUN apt-get update \
      git \
   && ln -s /usr/bin/python3 /usr/local/bin/python
 
-COPY . /opt/mythril
+
+COPY ./requirements.txt /opt/mythril/requirements.txt
```

Comment:
Yes, I've missed that. Is the Pipfile actually used/neeed?

Reply:
It doesn't look like it's needed but it might be helpful for devs for quick installation of the requirements, so i think it can be removed in the dockerfile.